cmd/zed/api/new/new.go

```diff
@@ -11,9 +11,9 @@ import (
 
 var NewSpec = &charm.Spec{
 	Name:  "new",
-	Usage: "new [spacename]",
-	Short: "create a new space",
-	Long: `The new command takes a single argument and creates a new, empty space
+	Usage: "new [poolname]",
```

Comment:
There are a bunch of things in the zpi command that should mirror the zed lake commands as we discussed (subsequent PR).

Reply:
Yep next thing I work on once this and the brim side changes get in.